torch/utils/hipify/hipify_python.py

```diff
@@ -709,6 +713,7 @@ def pattern(self):
 # Its mappings will trigger first, and only when a miss occurs will the lower-priority pytorch mapping take place.
 # When a file contains "sparse" in the filename, a mapping marked with API_SPARSE is preferred over other choices.
 PYTORCH_SPARSE_MAP = {}
+PYTORCH_SOLVER_MAP = {}
```

Comment:
Why is a separate `PYTORCH_SOLVER_MAP` needed here?
`PYTORCH_SPARSE_MAP` and `PYTORCH_SOLVER_MAP` should be unified and the comment describing this part of code updated.